docs/pages/server-access/guides/openssh.mdx

```diff
@@ -92,13 +97,8 @@ $ sudo bash -c "echo \"$KEY\" > /etc/ssh/teleport_user_ca.pub"
 $ sudo bash -c "echo 'TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub' >> /etc/ssh/sshd_config"
 ```
 
-Restart `sshd`. For systemd-enabled hosts, run the following command:
-
-```code
-$ sudo systemctl restart sshd
-```
-
-Now, `sshd` will trust users who present a Teleport-issued certificate.
+Now, `sshd` will trust users who present a Teleport-issued certificate after restarting
```

Comment:
Move this section to the next step then? It's odd to say "now" when you mean "in the future"